README.md

```diff
@@ -0,0 +1,27 @@
+## ConsoleDot Platform Common Middleware
+
+Common Go code for the `console.redhat.com` open-source platform.
+
+### Documentation
+
+None. Sorry :-) Read the code, this is a small repo. Also, `go doc` should work for most cases.
```

Comment:
Docs can be linked to from pkg.go.dev too https://pkg.go.dev/github.com/redhatinsights/platform-go-middlewares

Reply:
Oh good point, rebased.